content/en/docs/setup/best-practices/enforcing-pod-security-standards.md

```diff
@@ -0,0 +1,59 @@
+---
+reviewers:
+- tallclair
+- liggitt
+title: Enforcing Pod Security Standards
+weight: 40
+---
+
+<!-- overview -->
+
+This page provides an overview of best practices when it comes to enforcing [Pod Security Standards](/docs/concepts/security/pod-security-standards).
+
+<!-- body -->
+
+## Using the built-in Pod Security Admission Controller
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+The Pod Security Admission Controller intends to replace the deprecated PodSecurityPolicies. 
```

Comment:
I'd link to `/docs/reference/access-authn-authz/admission-controllers/#podsecurity` here.